example/Clarinet.toml

```diff
@@ -6,6 +6,14 @@ cache_dir = "./.cache"
 [[project.requirements]]
 contract_id = 'SP4SZE494VC2YC5JYG7AYFQ44F5Q4PYV7DVMDPBG.ststx-token'
 
+[[project.requirements]]
+contract_id = 'SP4SZE494VC2YC5JYG7AYFQ44F5Q4PYV7DVMDPBG.sip-010-trait-ft-standard'
+
+[contracts.rendezvous-token]
+path = 'contracts/rendezvous-token.clar'
+clarity_version = 2
+epoch = 2.5
```

Comment:
To make it easier for future contributors, could we add a comment here explaining why this contract has epoch 2.5?